packages/eslint-config-sdk/src/index.js

```diff
@@ -184,6 +184,15 @@ module.exports = {
         '@typescript-eslint/no-empty-function': 'off',
         '@sentry-internal/sdk/no-optional-chaining': 'off',
         '@sentry-internal/sdk/no-nullish-coalescing': 'off',
+
+        // Prevent permanent usage of `it.only`, `fit`, `test.only` etc
+        // We want to avoid debugging leftovers making their way into the codebase
+        "jest/no-focused-tests": "error",
```

Comment:
m: Should we maybe scope this to only `*.test.ts` and `*.test.js` files? May speed up linting slightly by only applying it to relevant files?

Reply:
Ahh yes we can probably limit it a little more in scope. I. missed that the `files` array also matches all *.ts/js files in the test sub directories.

Reply:
Done [ef9d16e](https://github.com/getsentry/sentry-javascript/pull/8489/commits/ef9d16e47f47098694681af4c4993251e03825b3)

Reply:
nice! But I guess you forgot to remove the rules from up top here 😅

Reply:
yup 🤦‍♂️ 😅 
(on the flipside, I realized while changing that we can also scope the entire plugin usage to just these files)